modules/claims_api/lib/bd/bd.rb

```diff
@@ -33,23 +34,24 @@ def search(claim_id, file_number)
     # Upload document of mapped claim
     #
     # @return success or failure
-    def upload(claim, pdf_path, file_number)
-      body = generate_upload_body(claim, pdf_path, file_number)
+    def upload(claim:, pdf_path:, file_number: nil)
+      @multipart = true
+      body = generate_upload_body(claim:, pdf_path:, file_number:)
       client.post('documents', body)&.body
```

Comment:
what if this fails? how does v1 handle that? and should we add some form of handling here?

Reply:
It's in a worker, so it'd retry it, should we not do that for this?